### PR TITLE
AD template review feedback part 1.

### PR DESCRIPTION
**Pull request by phenning**
@rustd @Tratcher 

Non middleware dependent changes as a result of the template review.

### CODE REVIEW

File: src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs [hunk 1]
```diff
@@ -75,16 +77,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
...
-            {
-                AutomaticAuthenticate = true
-            });
+            app.UseCookieAuthentication();
 
             app.UseOpenIdConnectAuthentication(new OpenIdConnectOptions
             {
                 ClientId = Configuration["Authentication:AzureAd:ClientId"],
                 Authority = Configuration["Authentication:AzureAd:AADInstance"] + "Common",
                 SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme,
```
Comment (Tratcher) [inline src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs]: Remove SignInScheme here, it takes it from services.

Comment (phenning) [inline src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs]: Updated PR

**Comment by Tratcher** [comment]
:shipit:

File: src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs [hunk 2]
```diff
@@ -75,16 +77,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
...
-                AutomaticAuthenticate = true
-            });
+            app.UseCookieAuthentication();
 
             app.UseOpenIdConnectAuthentication(new OpenIdConnectOptions
             {
                 ClientId = Configuration["Authentication:AzureAd:ClientId"],
                 Authority = Configuration["Authentication:AzureAd:AADInstance"] + "Common",
-                SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme,
+                CallbackPath = Configuration["Authentication:AzureAd:CallbackPath"],
```
Comment (rustd) [inline src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs:86]: What is the value of CallbackPath?

Comment (Tratcher) [inline src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs:86]: it should be `/signin-oidc` https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs#L52

**Comment by rustd** [comment]
I think that's what @phenning has. :shipit:

**Comment by phenning** [comment]
Yup, that is what the tooling will insert - "/signin-oidc"

**Comment by phenning** [comment]
merged

**Pull request closed without merging**